Maybe uninitialized variables when building with GCC in release mode

When building in release mode with GCC 7.3.0 (and probably other versions of GCC), several `maybe-uninitialized` errors are thrown. See below:

```shell
[ 84%] Building CXX object CMakeFiles/njoy21.dir/src/main.cpp.o
In file included from /Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/POWR/Lib3.hpp:14:0,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/POWR.hpp:9,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice.hpp:52,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/src/njoy21.hpp:20,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/src/main.cpp:1:
/Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/POWR/Lib3/Card12.hpp: In static member function 'static std::optional<std::vector<std::optional<std::pair<njoy::njoy21::lipservice::POWR::Lib3::Card12, std::pair<njoy::njoy21::lipservice::POWR::Lib3::Card12a, std::vector<std::pair<njoy::njoy21::lipservice::POWR::Lib3::Card12b, std::optional<njoy::njoy21::lipservice::POWR::Lib3::Card12c> > > > > > > > njoy::njoy21::lipservice::POWR::Lib3::readCard12(Istream&, std::vector<njoy::njoy21::lipservice::POWR::Lib3::Card5>, int) [with Istream = njoy::utility::stream::basic_RecordOrientedStream<std::basic_istream, char>]':
/Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/POWR/Lib3/Card12.hpp:1:7: error: '<anonymous>.njoy::njoy21::lipservice::POWR::Lib3::Card12::aw' may be used uninitialized in this function [-Werror=maybe-uninitialized]
 class Card12{
       ^~~~~~
/Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/POWR/Lib3/Card12.hpp:1:7: error: '<anonymous>.njoy::njoy21::lipservice::POWR::Lib3::Card12::temp' may be used uninitialized in this function [-Werror=maybe-uninitialized]
In file included from /Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/argument.hpp:4:0,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice.hpp:29,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/src/njoy21.hpp:20,
                 from /Users/jlconlin/NJOY21/Code/NJOY21/src/main.cpp:1:
/Users/jlconlin/NJOY21/Code/NJOY21/subprojects/lipservice/src/lipservice/argument/Type.hpp:17:8: error: '*((void*)(&<anonymous>)+16).njoy::njoy21::lipservice::argument::Type<njoy::njoy21::lipservice::POWR::Lib3::Card12::Fpa>::<anonymous>' may be used uninitialized in this function [-Werror=maybe-uninitialized]
 struct Type : public defaultField< policy::hasDefault< Policy > > {
        ^~~~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/njoy21.dir/src/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/njoy21.dir/all] Error 2
make: *** [all] Error 2
```

Note this does not occur when using the LLVM compiler.

--- a/ReleaseNotes.md
+++ b/ReleaseNotes.md
@@ -1,6 +1,9 @@
 # Release Notes&mdash;NJOY21
 Given here are some release notes for NJOY21. Each release is made through a formal [Pull Request](https://github.com/njoy/NJOY21/pulls) made on GitHub. There are links in this document that point to each of those Pull Requests, where you can see in great details the changes that were made. Often the Pull Requests are made in response to an [issue](https://github.com/njoy/NJOY21/issues). In such cases, links to those issues are also given.
 
+## pNJOY21 1.1.1](https://github.com/njoy/NJOY21/pull/107)
+This update fixes an issue where GCC would complain about a "maybe" uninitialized variable in [lipservice](https://github.com/njoy/lipservice). This only happens when compiled in release mode with GCC. Using the clang/LLVM compiler does not invoke this warning/error.
+
 ## [NJOY21 1.1.0](https://github.com/njoy/NJOY21/pull/104)
 This update moves the C++ standard to C++17; thus, a c++17-compliant compiler is needed to build NJOY21. Fortunately, C++17-compliant compilers have been available for several years.
 

--- a/ReleaseNotes.md
+++ b/ReleaseNotes.md
@@ -1,8 +1,10 @@
 # Release Notes&mdash;NJOY21
 Given here are some release notes for NJOY21. Each release is made through a formal [Pull Request](https://github.com/njoy/NJOY21/pulls) made on GitHub. There are links in this document that point to each of those Pull Requests, where you can see in great details the changes that were made. Often the Pull Requests are made in response to an [issue](https://github.com/njoy/NJOY21/issues). In such cases, links to those issues are also given.
 
-## pNJOY21 1.1.1](https://github.com/njoy/NJOY21/pull/107)
+## [NJOY21 1.1.1](https://github.com/njoy/NJOY21/pull/107)
 This update fixes an issue where GCC would complain about a "maybe" uninitialized variable in [lipservice](https://github.com/njoy/lipservice). This only happens when compiled in release mode with GCC. Using the clang/LLVM compiler does not invoke this warning/error.
+
+This update address issue #106.
 
 ## [NJOY21 1.1.0](https://github.com/njoy/NJOY21/pull/104)
 This update moves the C++ standard to C++17; thus, a c++17-compliant compiler is needed to build NJOY21. Fortunately, C++17-compliant compilers have been available for several years.

--- a/src/njoy21.hpp
+++ b/src/njoy21.hpp
@@ -17,7 +17,11 @@
 #include "dimwits.hpp"
 #include "Log.hpp"
 #include "utility.hpp"
-#include "lipservice.hpp"
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+  #include "lipservice.hpp"
+#pragma GCC diagnostic pop
 
 #include "njoy_c.h"
 

--- a/src/njoy21/Version.hpp
+++ b/src/njoy21/Version.hpp
@@ -4,7 +4,7 @@ public:
   // Change whenever new capability is implemented
   static constexpr int minorVersion{1};
   // Change whenever merge to master branch is done
-  static constexpr int patchVersion{0};
+  static constexpr int patchVersion{1};
 
   static std::string version(){
     return std::to_string( majorVersion ) + "." +

